data/core/macros/special-notes.cfg

```diff
@@ -299,134 +249,128 @@ _"This unit has a defense cap on certain terrain types — it cannot achieve a h
     [/special_note]
 #enddef
 
-#define NOTE_DIVERSION
-    [special_note]
-        note=_"If there is an enemy of the target on the opposite side of the target while attacking it, this unit diverts the target's attention and reduces its chance to hit."
-    [/special_note]
-#enddef
-
 # Deprecated versions here
 
 #define SPECIAL_NOTES_SPIRIT
-#deprecated 1 Use {NOTE_SPIRIT} instead (note the singular), which generates a [special_note] tag.
+#deprecated 3 1.17 Use {NOTE_SPIRIT} instead (note the singular), which generates a [special_note] tag.
 {INTERNAL:SPECIAL_NOTES_SPIRIT}#enddef
 
 #define SPECIAL_NOTES_ARCANE
-#deprecated 1 Use {NOTE_ARCANE} instead (note the singular), which generates a [special_note] tag.
+#deprecated 3 1.17 Use {NOTE_ARCANE} instead (note the singular), which generates a [special_note] tag.
```

Comment:
This seems closer to `DEP_LEVEL::PREEMPTIVE`, a simple maintenance wrapper can be left in the code indefinitely and just means text appears in a different place on the help page.

The thing that's prompted me to look at this is that 3000 errors are reported when rebuilding the cache with Ageless Era installed. `DEP_LEVEL::PREEMPTIVE` would mean they were only warnings and that they would only be reported in the 1.17 branch, but `DEP_LEVEL::FOR_REMOVAL` means they'll be errors and reported in the 1.16 branch.

@ProditorMagnus: heads-up.

Reply:
Looks like simple case of removing all notes which show error.

Reply:
Technically, leaving this in now means the text will be shown twice on the help page, but that's still just a cosmetic issue, so sure.

What are these errors you're referring to? I didn't remove any macros, so there shouldn't be any errors; at most there should be tons of deprecation warnings. Are you referring to those as errors?

Reply:
This doesn't seem to show the text twice, and the deprecation message says that it should be replaced with `{NOTE_ARCANE}` rather than removed.

By errors and warnings, I mean the difference between `lg::warn()` and `lg::err()`.
https://github.com/wesnoth/wesnoth/blob/faac6939f42a86cd81b79cba20104d4ef58b7ad8/src/deprecation.cpp#L42-L56

Reply:
Just noting that if you were to leave in the old `SPECIAL_NOTES` macro, you _would_ get the text displaying twice, as this example shows.

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/578731/119283467-308ad880-bc0b-11eb-95fc-f6658fdc5c16.png">

Reply:
That suggests the deprecation level on the `SPECIAL_NOTES` macro itself is too low. But that example is for one of the auto-detected abilities, not `{SPECIAL_NOTES_ARCANE}`.

Reply:
Right, in that case, you'd only get a double note if you actually included both `{SPECIAL_NOTES_ARCANE}` and `{NOTE_ARCANE}`.

Reply:
The reason that I haven't marked this resolved is that I still think the DEP_LEVEL on `SPECIAL_NOTES` ought to be higher.